addons/clickhouse/templates/cmpd-clickhouse.yaml

```diff
@@ -128,7 +133,7 @@ spec:
       valueFrom:
         componentVarRef:
           compDef: ch-keeper-24
```

Comment:
check if compDef here should use regex pattern match

Reply:
fixed